src/components/home/success-stories-overview.astro

```diff
@@ -7,7 +7,9 @@ import { Image } from 'astro:assets';
 import { Spacer } from '@/components/ui/spacer';
 import FormattedDate from '@/components/blog/formatted-date.astro';
 
-const stories = await getCollection('successStories');
+const sortByDate = (a: any, b: any) =>
```

Comment:
will clean this up in a subsequent pass (as part of consolidating the collections).